### PR TITLE
Assignment2 : Find destination city

### PR DESCRIPTION
**Pull request by deepakghadge19**
Added code for assingment2 : find destination city

### CODE REVIEW

File: src/secondAssignment_14March2023/FindDestinationCity.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class FindDestinationCity {
+
+	public static void main(String[] args) {
+		String input[][] = { { "London", "Newyork" }, { "Newyork", "Lima" }, { "Lima", "Sao Paulo" } };
+		// String input[][] = { { "B", "C" }, { "D", "B" }, { "C", "A" } };
+		ArrayList<String> AL = new ArrayList<>();
```
Comment (amitsahu8) [inline src/secondAssignment_14March2023/FindDestinationCity.java:12]: variable name is not proper

File: src/secondAssignment_14March2023/FindDestinationCity.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+		for (int i = 0; i < input.length; i++) {
```
Comment (amitsahu8) [inline src/secondAssignment_14March2023/FindDestinationCity.java:13]: create a new function and use it in main

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: src/secondAssignment_14March2023/FindDestinationCity.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package secondAssignment_14March2023;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class FindDestinationCity {
+
+	public static void main(String[] args) {
```
Comment (amitsahu8) [inline src/secondAssignment_14March2023/FindDestinationCity.java:9]: remove .class file and work on the coding guideline points

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: src/secondAssignment_14March2023/FindDestinationCity.java (hunk 3 above)
Comment (deepakghadge19) [reply src/secondAssignment_14March2023/FindDestinationCity.java]: Done

File: src/secondAssignment_14March2023/FindDestinationCity.java (hunk 2 above)
Comment (deepakghadge19) [reply src/secondAssignment_14March2023/FindDestinationCity.java]: Done

File: src/secondAssignment_14March2023/FindDestinationCity.java (hunk 1 above)
Comment (deepakghadge19) [reply src/secondAssignment_14March2023/FindDestinationCity.java]: Done

File: src/secondAssignment_14March2023/FindDestinationCity.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class FindDestinationCity {
+    public void destinationCity(String[][] input) {
+        ArrayList<String> city = new ArrayList<>();
+        for (int row = 0; row < input.length; row++) {
+            for (int column = 0; column < 2; column++) {
```
Comment (amitsahu8) [inline src/secondAssignment_14March2023/FindDestinationCity.java:11]: why 2? it should be dynamic

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: src/secondAssignment_14March2023/FindDestinationCity.java (hunk 4 above)
Comment (deepakghadge19) [reply src/secondAssignment_14March2023/FindDestinationCity.java]: Done!! Added code to calculate length of column.

**Review by amitsahu8 — APPROVED** [review]

**Review by rohitlamsonge66 — APPROVED** [review]

**Pull request merged**